scripts/download-contributor-data.js

```diff
@@ -0,0 +1,45 @@
+import fs from 'fs';
+
+const apiUrl = 'https://api.github.com/repos/appwrite/appwrite/contributors';
+const perPage = 100;
+
+const outputFile = `./src/lib/contributors.ts`;
+
+async function fetchContributors() {
+    let page = 1;
+    let contributorsData = [];
+    let hasMoreData = true;
+
+    while (hasMoreData) {
+        console.log(page);
```

Comment:
is this useful?

Reply:
The log? Just to keep track of how progress is going.

Reply:
Can we make the log more meaningful instead of just numbers 😁